nuodb.go

```diff
@@ -168,13 +169,26 @@ func (c *Conn) Begin() (driver.Tx, error) {
 }
 
 func (c *Conn) Exec(sql string, args []driver.Value) (driver.Result, error) {
+	if len(args) > 0 {
+		return nil, driver.ErrSkip
+	}
+	return c.ExecContext(context.Background(), sql, nil)
+}
+
+func (c *Conn) ExecContext(ctx context.Context, sql string, args []driver.NamedValue) (driver.Result, error) {
```

Comment:
Perhaps document which driver Stmt interfaces are now implemented.

Something like this next to the Stmt struct definition:

```go
var _ interface {
	driver.Stmt
	driver.StmtQueryContext
	// driver.StmtExecContext
} = (*Stmt)(nil)
```

Reply:
Good idea. Done.